.eslintrc

```diff
@@ -6,6 +6,7 @@
     "plugin:@typescript-eslint/eslint-recommended",
     "plugin:@typescript-eslint/recommended",
     "plugin:import/typescript",
+    "plugin:no-typeof-window-undefined/recommended",
```

Comment:
Not sure if it's better to specify this plugin config (with a single rule) or specify it in plugins and enable this rule specifically.